workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Partials/Improbable.Coordinates

```diff
@@ -0,0 +1,166 @@
+public static readonly Coordinates ZERO = new Coordinates(0, 0, 0);
+
+public static Vector3d operator -(Coordinates v1, Coordinates v2)
+{
+    return new Vector3d(v1.X - v2.X, v1.Y - v2.Y, v1.Z - v2.Z);
```

Comment:
Should this return a `Coordinates`?

Reply:
The idea is that coordinate is an absolute and vector3d is a diff. I personally hate this though.

Reply:
I think we should probably change this to coordinate - coordinate = coordinate. Once we have custom codegen I would expect a different type to take this job over.